tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -248,12 +249,13 @@ void onRenderTaskPostExecute() {
     if( mTileRenderListener != null ) {
       mTileRenderListener.onRenderComplete();
     }
-    invalidate();
-    requestRender();
```

Comment:
While it is indeed fast-fail out, I believe that if a new render is need since a change to the tiles to be displayed, this is the responsibility of `TileView`. For example, we have a `requestRender` at the end of a pan, at a `DetailLevel`change, etc.
Letting a `requestRender` here just causes permanent extra computation with ` DetailLevel.computeCurrentState()`. I don't know the impact on cpu usage (and thus on battery life), but if that can be avoided, why not do it?